src/gpl/src/initialPlace.h

```diff
@@ -42,6 +42,11 @@ namespace utl {
 class Logger;
 }
 
+struct tri{
```

Comment:
Good to have a solid comment on the struct definition for what it's representing / how it's generally used. Looks like maybe the convention in this code base is to use a leading cap, e.g. `Tri` ?

Reply:
Use meaningful names

Reply:
I can delete this structure actually .... I didn't use it in the final result.